ocaml/testsuite/tests/typing-misc/labels.ml

```diff
@@ -104,7 +104,7 @@ val f : (?x:int -> unit -> int) -> int = <fun>
 Line 1, characters 46-47:
 1 | let f g = ignore (g : ?x:int -> unit -> int); g ();;
                                                   ^
-Warning 19 [non-principal-labels]: eliminated optional argument without principality.
+Warning 19 [non-principal-labels]: eliminated omittable argument without principality.
```

Comment:
To confirm, it's okay that this message is changed as long as the warning flag is the same?

Reply:
Yes, it’s fine. Thanks for checking.